api-platform/core/3.3/config/packages/api_platform.yaml

```diff
@@ -0,0 +1,18 @@
+api_platform:
+    title: Hello API Platform
+    version: 1.0.0
+    formats:
+        jsonld: ['application/ld+json']
+    docs_formats:
+        jsonld: ['application/ld+json']
+        jsonopenapi: ['application/vnd.openapi+json']
+        html: ['text/html']
+    defaults:
+        stateless: true
+        cache_headers:
+            vary: ['Content-Type', 'Authorization', 'Origin']
+        extra_properties:
+            standard_put: true
+            rfc_7807_compliant_errors: true
+    keep_legacy_inflector: false
+    use_symfony_listeners: true
```

Comment:
@dunglas should we keep this to true? If we continue to recommend using controllers we should keep this to true, if not maybe that we should recommend using false instead?

Reply:
I would set it to false by default, but I've no strong opinion about this.